docs/WebEditorBasics/WebPlayMode.md

```diff
@@ -0,0 +1,53 @@
+---
+id: WebPlayMode
+title: Web Play Mode
+---
+
+Don't always have access to a device? Waiting for a battery to recharge? No problem!
+
+You can continue to build with Enklu Cloud's blazing fast iteration times by using Web Play Mode to test your experiences.
+
+// TODO: Liam's awesome gif here.
+
+## What is Web Play Mode?
```

Comment:
probably don't need this title, since the text below goes on to define web play mode